docs/release-payload-controllers.md

```diff
@@ -0,0 +1,73 @@
+
+# release-payload-controller
+The `release-payload-controller` is a single binary that contains all the controllers that are responsible for handling all the intereactions with `ReleasePayload` objects. 
+
+## release-controller
+The `release-controller` is responsible for creating all `ReleasePayload` objects.
+
+## release-payload-status-controller
+```
+Writes `.status`
+```
+* Do we need this if there isn't anything that needs to be written initially when a `ReleasePayload` is created?
+
+## release-payload-created-controller
+```
+writes: .status.condition.type=PayloadCreated
+reads: release creation job
+```
+### pseudocode
+```
+if job is successful, then PayloadCreated=true
+else, PayloadCreated=false with status of the job 
+```
+
+## release-payload-accepted-controller
+```
+writes: .status.condition.type=PayloadAccepted
+reads: status.jobstatus, spec.payloadOverride.override
+```
+### pseudocode
+```
+if (spec.payloadOverride.override == "Accepted"), then PayloadAccepted=true
+if status.blockingJobResults.state == "Success", then PayloadAccepted=true
+else, PayloadAccepted=false with a list of jobs that need to pass as a reason
```

Comment:
before the else.  If there are no blockingjobResults, then the condition is false.  This means that the thing creating the list of blocking jobs needs to create that list all at once.